utils/apply-settings.sh

```diff
@@ -93,7 +93,7 @@ fi
 apply-setting "$lgsm_cfg" maxplayers "maxplayers=$maxplayers"
 
 # Custom Map Support
-function start-custom-map-server() (
+function start-custom-map-server() {
```

Comment:
This is actually valid bash and not a typo.  I have since fixed the problem with apply-settings.sh.  For reference here's the following documentation so you know I'm not just saying something untrue.

* https://www.gnu.org/software/bash/manual/bash.html#Shell-Functions
* https://www.gnu.org/software/bash/manual/bash.html#Compound-Commands

Function definition
```
function fname [()] compound-command [ redirections ]
```

A function can have any compound command.  That means `{}` (which is how you normally see functions) is a valid function syntax.  However also,

* Compound command for subshell `()` is valid.
* Any other compound command is valid in a function definition.

The following three function definitions in bash are all valid.

```bashsed

# valid option one
function foo() {
    (
        for x in 1 2 3; do
            echo "$x"
        done
    )
}

# valid option two with subshell compound command
function foo() (
    for x in 1 2 3; do
        echo "$x"
    done
)

# valid option 3 using for loop compound command
function foo() for x in 1 2 3; do
    echo "$x"
done
```